mmaction/models/backbones/st_gcn.py

```diff
@@ -0,0 +1,449 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import BACKBONES
+
+
+def zero(x):
+    return 0
+
+
+def iden(x):
+    return x
+
+
+def get_hop_distance(num_node, edge, max_hop=1):
```

Comment:
- zero
- iden
- get_hop_distance
- normalize_graph
- Graph

can be placed in skeletongcn/utils.py